bindings/go/generator/ocmtypegen/test/type.go

```diff
@@ -0,0 +1,12 @@
+package test
+
+import (
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+// +k8s:deepcopy-gen=true
+//
+//go:generate ocmtypegen $GOFILE
```

Comment:
What is `$GOFILE` about?

Initially I thought `$GOFILE` should actually be a go-file, e.g. `filename.go` (based on the first `log.Fatal` from the `main` method). But seeing it here as well confuses me. 

Isn't obvious that it is a `$GOFILE` through the file extension `.go`?

Reply:
Nevermind, `$GOFILE` is a variable set by `go generate` and contains the base name of the file